docs/tutorials/development.md

```diff
@@ -137,6 +137,86 @@ func (f *myFilter) Response(ctx filters.FilterContext) {
 
 Find a detailed example at [how to develop a filter](../reference/development.md#how-to-develop-a-filter).
 
+### Filters with cleanup
```

Comment:
I am not sure we should document it here at this time as this might create confusion for the users.
This change only closes filters on the routing shutdown, i.e. once in the Skipper lifetime while filters could be recreated on each route update. Users may get confused and think Close will be called for each their filter instance that is not used anymore.

The reason it works for admission control is because there is a postprocesor that closes unused filters (which technically is still incorrect as there might be in-flight requests during route update)

I.e. it does not solve https://github.com/zalando/skipper/issues/202 but only makes a small step towards.

Reply:
good point about the incorrect closing. Maybe we should introduce a closing registry which will wait for timeouts like ResponseHeaderTimeout long before it closes the filters and we move in the PostProcessor the "Close" call to something like closingRegistry.Add(filterInstance). wdyt?

Reply:
a sync.Atomic as gauge per route?